types/cucumber/index.d.ts

```diff
@@ -7,8 +7,11 @@
 //                 ErikSchierboom <https://github.com/ErikSchierboom>
 //                 Peter Morlion <https://github.com/petermorlion>
 //                 Don Jayamanne <https://github.com/DonJayamanne>
+//                 Alberto Silva <https://github.com/albertossilva>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.4
+// TypeScript Version: 3.2
```

Comment:
Once you get rid of the default type parameter on setWorldConstructor, I think you can set this back to 2.4.